src/function/index.ts

```diff
@@ -2,3 +2,4 @@ export { Context as HandlerContext } from './context.js'
 export { Event as HandlerEvent } from './event.js'
 export { BuilderHandler, Handler, BackgroundHandler, HandlerCallback, StreamingHandler } from './handler.js'
 export { BuilderResponse, Response as HandlerResponse, StreamingResponse } from './response.js'
+export * from './v2.js'
```

Comment:
Any reason this isn't a named export?

Reply:
No particular reason, no. I put the named exports inside of `v2.js`, but I can also move that up if you think that's better.